RELEASES.md

```diff
@@ -1,3 +1,128 @@
+Version 1.75.0 (2023-12-28)
+==========================
+
+<a id="1.75.0-Language"></a>
+
+Language
+--------
+
+- [Stabilize `async fn` and return-position `impl Trait` in traits.](https://github.com/rust-lang/rust/pull/115822/)
+- [Allow function pointer signatures containing `&mut T` in `const` contexts.](https://github.com/rust-lang/rust/pull/116015/)
```

Comment:
I dunno if this is fixable but when skimming the notes I saw "`&mut T` in `const` contexts" and got momentarily confused/excited until I re-read the start of the line.